guides/common/assembly_accessing-server.adoc

```diff
@@ -8,11 +8,11 @@ endif::[]
 
 include::modules/proc_logging-in.adoc[leveloffset=+1]
 
-include::modules/proc_using-freeipa-credentials-to-log-in-to-the-project-hammer-cli.adoc[leveloffset=+1]
+include::modules/proc_logging-in-to-hammer-cli-with-freeipa-credentials.adoc[leveloffset=+1]
 
-include::modules/proc_using-freeipa-credentials-to-log-in-to-the-ProjectWebUI-with-a-Firefox-browser.adoc[leveloffset=+1]
+include::modules/proc_logging-in-to-the-projectwebui-with-freeipa-credentials-in-mozilla-firefox.adoc[leveloffset=+1]
 
-include::modules/proc_using-freeipa-credentials-to-log-in-to-the-ProjectWebUI-with-a-Chrome-browser.adoc[leveloffset=+1]
+include::modules/proc_logging-in-to-the-projectwebui-with-freeipa-credentials-in-chrome.adoc[leveloffset=+1]
```

Comment:
It's odd to me that one is "Mozilla Firefox" and the other "Chrome" instead of "Google Chrome".